virt/virt-2-5-release-notes.adoc

```diff
@@ -38,6 +38,12 @@ The SVVP Certification applies to:
 
 //CNV-7160 - New virtctl commands to expose raw qemu guest agent data
 
+* {VirtProductName} {VirtVersion} adds three new xref:../virt/virt-using-the-cli-tools.adoc#virt-virtctl-commands_virt-using-the-cli-tools[virtctl commands] to manage qemu guest agent data:
```

Comment:
Markup "virtctl" in backticks.

Reply:
Also seems like "qemu" is usually capitalized "QEMU" elsewhere.

Reply:
Where should virtctl be marked up? 

Doing it in the xref would break the xref, in the xref display text would show the backticks, the story title is already commented out, and the bulleted list has the full command in backticks rather than just the virtctl part.

Capitalizing QEMU done.